pkgs/tools/misc/lerpn/default.nix

```diff
@@ -0,0 +1,23 @@
+{ python3, lib, fetchFromGitea }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "lerpn";
+  version = "unstable-2023-06-08";
+
+  src = fetchFromGitea {
+    domain = "gitea.alexisvl.rocks";
+    owner = "alexisvl";
+    repo = pname;
+    rev = "df792ae930005415afd4e56950de671cdd23fb26";
+    sha256 = "sha256-XfQssik8POBgfCbmP3tjUGomE769yrujqJhImd6vxTQ=";
+  };
+
+  dontUseSetuptoolsCheck = true; # doesn't halt
+
```

Comment:
Would you please add `checkPhase`?
The upstream has a test script.

and it is preferable to add `pythonImportsCheck` for a quick check.